torch/quantization/fuse_modules.py

```diff
@@ -1,8 +1,18 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import torch
+import copy
 
 import torch.nn._intrinsic.modules.fused as torch_fused
+def fuse_conv_relu(conv, relu):
```

Comment:
you don't need these function - just put torch.nn._intrinsic.ConvReLU2d in the dict below